### PR TITLE
Update raml title and baseUri

### PR DESCRIPTION
**Pull request by iwanbk**
Update the title from `User/Group example api` to `Metadata`.

Remove trailing slash from .raml `baseUri`

### CODE REVIEW

File: api_spec/main.raml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
-title: User/Group example api
+title: Metadata
```
Comment (zaibon) [inline api_spec/main.raml]: 0-Metadata

File: api_spec/main.raml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 version: 0.8.0
-baseUri: 'http://127.0.0.1:5000/'
+baseUri: 'http://127.0.0.1:5000'
```
Comment (zaibon) [inline api_spec/main.raml]: is this the default port used by the server ?

File: api_spec/main.raml (hunk 1 above)
Comment (iwanbk) [reply api_spec/main.raml]: fixed

File: api_spec/main.raml (hunk 2 above)
Comment (iwanbk) [reply api_spec/main.raml]: Unfortunately i don't know, you and @lafont-e know better

**Pull request merged**